src/platform/utilities/feature-toggles/featureFlagNames.js

```diff
@@ -29,4 +29,5 @@ export default Object.freeze({
   vaViewDependentsAccess: 'vaViewDependentsAccess',
   allowOnline1010cgSubmissions: 'allow_online_10_10cg_submissions',
   gibctEybBottomSheet: 'gibctEybBottomSheet',
+  routeStLouisRPOtoBuffaloRPO: 'route_st_louis_rpo_to_buffalo_rpo',
```

Comment:
Dumb question: If the values in this object use both camelcase and snakecase, how do you decide which one to use? Does it matter?